standards/rmrk2.0.0/interactions/baseeq.md

```diff
@@ -0,0 +1,65 @@
+# BASEEQ

```

Comment:
Equippable. Was saving bytes. Open to suggestions.

Reply:
I am not sure but if we are to add generic SET VALUE interaction for NFTs we can re-use it for BASE, and the same for CHANGEISSUER maybe we can simply reuse it for BASE by passing some kind of type: base in there?

Reply:
Good thinking, can be incorporated.

Reply:
So this can be like that perhaps, where `keys` can be an array to target nested paths;
And `type` is either `'NFT' | 'BASE'` 

`0x{bytes(RMRK::SETVALUE::{version}::{type}::{id}::{keys}::{+/-?values|*})}`
`RMRK::BASEEQ::2.0.0::BASE::kanaria-epic-birds::[parts, wing_slot_1]::0aff6865bed3a66b-DLEP`

And then 

`0x{bytes(RMRK::CHANGEISSUER::{version}::{type}::{id}::{newissuer})}`.
`rmrk::CHANGEISSUER::2.0.0::kanaria-epic-birds::BASE::HviHUSkM5SknXzYuPCSfst3CXK4Yg6SWeroP6TdTZBZJbVT`

Reply:
Looking at Statemine uniques there's `setAttribute` which accepts key and value, maybe we can mimic it

https://polkadot.js.org/docs/substrate/extrinsics/#setattributeclass-compactclassid-maybe_instance-optioninstanceid-key-bytes-value-bytes

Reply:
Decided to make BASEEQ->EQUIP to be clearer, but separate from SET since it's a very specific op. Will rename set to setattribute to match Uniques

Reply:
Cool, yeah I was worried about nested obejct with this SET, so better to have one specific OP